en/news/_posts/2017-09-18-ruby-2-5-0-preview1-released.md

```diff
@@ -43,7 +43,7 @@ See [NEWS](https://github.com/ruby/ruby/blob/v2_5_0_preview1/NEWS)
 or commit logs for details.
 
 With those changes,
-[6162 files changed, 28537 insertions(+), 339582 deletions(-)](https://github.com/ruby/ruby/compare/v2_4_0...v2_5_0_preview1)
+[6162 files changed, 339582 insertions(+), 28537 deletions(-)](https://github.com/ruby/ruby/compare/v2_4_0...v2_5_0_preview1)
```

Comment:
Thanks for the fix! 
The numbers I'm seeing are:
https://github.com/ruby/ruby/compare/v2_4_0...v2_5_0_preview1
6,162 changed files with 339,744 additions and 28,699 deletions.

🤔

Reply:
You're right, might be they changed since the article has been written. Updating nevertheless